Dockerfile.fluentbit

```diff
@@ -1,9 +1,8 @@
 FROM registry.access.redhat.com/ubi7/ubi-minimal
 ARG VERSION
-RUN \
-  echo -e '[td-agent-bit]\nname=td-agent-bit\nbaseurl=https://packages.fluentbit.io/centos/7/$basearch' >/etc/yum.repos.d/td-agent-bit.repo && \
-  rpm --import https://packages.fluentbit.io/fluentbit.key && \
-  microdnf update && \
-  microdnf install td-agent-bit-$VERSION && \
-  microdnf clean all
+RUN echo -e '[td-agent-bit]\nname=td-agent-bit\nbaseurl=https://packages.fluentbit.io/centos/7/$basearch' >/etc/yum.repos.d/td-agent-bit.repo && \
+    rpm --import https://packages.fluentbit.io/fluentbit.key
+RUN microdnf update
+RUN microdnf install td-agent-bit-$VERSION
+RUN microdnf clean all
```

Comment:
What is the purpose of splitting RUN statements here? This will create additional layers and it is recommended to execute update, install, and clean within the same layer in this case.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
It is recommended but when you are debugging/working with images all the process takes ages to rerun when we have n commands chained. And final result is still +/- same. We are not here to produce perfect image layer :) 
But agree, you are right. It just for convenience.

Reply:
I see that we have `--no-cache` in the Makefile for building the images, so that will take care of rerunning all layers for the final build.
The note is not to use Makefile then to take advantage of caching when building locally for dev the way you have it changed. Do you build without Makefile action currently?